source/frontend/src/components/common/form/TableSelect.tsx

```diff
@@ -21,81 +18,35 @@ type RequiredAttributes<T extends object> = {
   selectedTableHeader: React.FC<React.PropsWithChildren<ISelectedTableHeaderProps>>;
   /** The columns that should be used for the secondary, "saved items" table */
   columns: ColumnWithProps<T>[];
-  /** child component that handles selection */
-  children: ReactNode;
+  onRemove: (items: T[]) => void;
 };
 
-type OptionalAttributes = {
```

Comment:
dead code, first half of tableSelect no longer exists, using the contact picker instead.

Reply:
+1